modules/programs/tmux.nix

```diff
@@ -199,10 +453,10 @@ in
 
       shortcut = mkOption {
         default = defaultShortcut;
-        example = "a";
+        example = "C-a";
```

Comment:
Should this example be also stateVersion dependent?

Reply:
Already done this.

Reply:
Not for the example from what I see.